lib/DockerWrapper.js

```diff
@@ -8,6 +8,7 @@ const uuidv1 = require('uuid/v1')
 const slugify = require('slugify');
 const tar = require('tar-fs');
 const moment = require('moment');
+const errHandler = e => console.log(e);
```

Comment:
Should the errHandler also return an error response to the client?

Reply:
Unfortunately we've already returned a response by the time a `.catch` would be handled. Unless we move when we send the response back in the process of creating the node.

I just created this function as a potential handler we could reuse and maybe build upon later where we might find a way to asynchronously send a message to the client(s).